uaclient/util.py

```diff
@@ -619,9 +619,10 @@ def _subp(
             stderr=err.decode("utf-8"),
         )
     if capture:
+        redacted_cmd = redact_sensitive_logs(" ".join(args))
```

Comment:
I think we actually need to pull this up above the [try: proc = subprocess.Popen](https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/util.py#L605) so that we can make sure even if there are errors we don't leak that token in [line 613](https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/util.py#L613) and  [line 616](https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/util.py#L616)

Reply:
Oh yes, makes sense.